xAPI.md

```diff
@@ -302,7 +302,44 @@ As a rule of thumb, if the guideline appears technical or seems to be a requirem
 as such. This is especially true of longer, more, detailed explanations and of tables, each of which would 
 be unintuitive and/or lengthy to dissect into a list of requirements.
 
-## 3.0 Guidenelines for Implementing Technical Specificationss Like xAPI
+## 3.0 Guidelines for Implementing Technical Specifications Like xAPI
```

Comment:
Thinking about it, this title is quite odd. I see fairly commonly (and like) titles such as "How To Read This Specification". What do ppl think about that as a replacement?